tf2_bullet/package.xml

```diff
@@ -21,5 +21,10 @@
   <run_depend>bullet</run_depend>
   <run_depend>geometry_msgs</run_depend>
 
-</package>
+  <depend>bullet</depend>
```

Comment:
I have include this line for rosdep. But it's not working, "bullet" is still not installed

Reply:
Yeah.  This is necessary for this to work with `rosdep` and the buildfarm, so it is necessary.  Unfortunately the CI system doesn't use `rosdep` to calculate this stuff, so we still have to modify the CI scripts to install it manually.  There are a few steps to getting this in:

1.  Figure out a way to install bullet on Windows.  Ideally this would be a chocolatey package.
    1.  If a chocolatey package is available, edit https://github.com/ros2/ros2_documentation/blob/master/source/Installation/Foxy/Windows-Development-Setup.rst to add it.  Also log into each of the CI Windows machines and install it.
    1.  If a chocolatey package is not available there are 2 options: build and host a chocolatey package ourselves (we have done this for some dependencies), or vendor bullet (this is more common).  One example of a vendored package is https://github.com/ros2/yaml_cpp_vendor; that can be used as a template.
1.  Add libbullet-dev to https://github.com/ros2/ci/blob/25a3bdf20a310b1df8b28a5af92e1eb66d5681ea/linux_docker_resources/Dockerfile#L40 (that will fix it for Linux).
1.  Edit https://github.com/ros2/ros2_documentation/blob/master/source/Installation/Foxy/OSX-Development-Setup.rst , add a line to `brew install bullet`.
1.  Log into each of the CI machines and run `brew install bullet`.

In this case the most important thing to figure out is how we are going to deal with it for Windows, since I think it is pretty straightforward for both Linux and macOS.  Once we have that figured out the rest is just work that needs to get done.